app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -265,7 +265,6 @@ private void promptSpeechInput() {
     }
 
 
-
```

Comment:
Make sure only your changes get in. It's hard for reviewing these spacing and indentation changes.